ocs_ci/deployment/deployment.py

```diff
@@ -469,8 +478,13 @@ def deploy_ocs_via_operator(self, image=None):
         self.subscribe_ocs()
         operator_selector = get_selector_for_ocs_operator()
         subscription_plan_approval = config.DEPLOYMENT.get("subscription_plan_approval")
+        ocs_version = config.ENV_DATA["ocs_version"]
+        if Version.coerce(ocs_version) >= Version.coerce("4.9"):
+            ocs_operator_name = defaults.ODF_OPERATOR_NAME
+        else:
+            ocs_operator_name = defaults.OCS_OPERATOR_NAME
         package_manifest = PackageManifest(
-            resource_name=defaults.OCS_OPERATOR_NAME,
+            resource_name=ocs_operator_name,
```

Comment:
I think we will need to make some more changes to verify also ODF CSV.  But not 100% sure about that. I think we will have CSV for ODF + CSV for OCS so we will need to verify both in the case of 4.9. But let's see once we will have the build how it's exactly looking.  So let's wait with this implementation once we have working 4.9 build with dependencies and ODF operator.

Line 505:
```
csv.wait_for_phase("Succeeded", timeout=720)
```